client/src/components/formlabels/FormLabels.tsx

```diff
@@ -23,17 +23,20 @@
  *  FormLabels components.
  */
 import * as React from "react";
```

Comment:
We can modernize imports, maybe:
```suggestion
import React from "react";
```